environments/stfc-base/inventory/group_vars/all/user_cluster_k8s_app_configs.yml

```diff
@@ -0,0 +1,24 @@
+azimuth_capi_operator_app_templates_jupyterhub_default_values:
+  jupyterhub:
+    singleuser:
+      profileList:
+      - display_name: "Python environment (minimal)"
+        description: "Minimal Python environment"
+        default: true
+        kubespawner_override:
+          image: quay.io/jupyter/minimal-notebook:latest
+      - display_name: "Data Science Notebook"
+        description: "Libraries for data analysis in Python, R and Julia."
+        kubespawner_override:
+          image: quay.io/jupyter/datascience-notebook:latest
+      - display_name: "GPU-enabled Machine Learning environment"
+        description: >
+          Support for GPU-enabled machine learning in Python.
+          JupyterLab servers of this type will only start on node groups
+          including nodes that have access to an NVIDIA GPU.
+        kubespawner_override:
+          image: cschranz/gpu-jupyter:1.9_cuda-12.6_ubuntu-24.04_python-only # TODO: Is Cuda 12.6 compatibile with all our GPU flavors?
+          node_selector: {'nvidia.com/gpu.present': 'true'} # TODO: Does NFD or gpu-operator properly apply this label?
+      # TODO: Add custom image option, take details from azimuth-ui.schema.yaml
+      # Might need to fork for that...
```

Comment:
Can we drop the TODO on cuda 12.6, this is compatible:
https://github.com/stfc/cloud-docker-images/blob/master/notebooks/jupyter-ml-notebook/Dockerfile#L1

For the second TODO `gpu-operator` adds the label once the GPU + drivers are up and validated, so we can drop the todo here too